apps/ledger-live-desktop/src/renderer/components/SpendableAmount.tsx

```diff
@@ -56,6 +56,7 @@ const SpendableAmount = <T extends TransactionCommon>({
       showAllDigits={showAllDigits}
       showCode
       alwaysShowValue
+      data-test-id="modal-spendable-banner"
```

Comment:
This is the same test id as below. Should it not be `modal-spendable-amount`?

Reply:
my mistake, the one in spendable banner is unnecessary.